Monika After Story/game/script-ch30.rpy

```diff
@@ -913,7 +913,8 @@ label spaceroom(start_bg=None, hide_mask=None, hide_monika=False, dissolve_all=F
                 #     force_exp = "monika idle"
 
             if not renpy.showing(force_exp):
-                renpy.show(force_exp, tag="monika", at_list=[t11], zorder=MAS_MONIKA_Z)
+                # please keep i11 - otherwise monika jumps if we scene change without dissolving all
+                renpy.show(force_exp, tag="monika", at_list=[i11], zorder=MAS_MONIKA_Z)
```

Comment:
With these changes Monika changes her position after the greeting, moves to the right. I think that happens because `i11` and `t11` actually set different positions. And it is noticeable.